### ISSUE TITLE
DetectedLongRunningApplicationTask warning on each websocket connection

### ISSUE THREAD

**Issue by standsed** [Azure/azure-signalr#1346]
### Describe the bug
All connections (using only websockets) to Azure SignalR hub causes DetectedLongRunningApplicationTask warning.
Looks like it was introduced with https://github.com/Azure/azure-signalr/pull/1332.

### To Reproduce
Repro with server and client here https://github.com/standsed/SignalRRepro
Please change ConnectionString in Startup.cs to your Azure SignalR instance.

### Exceptions (if any)
```
warn: Microsoft.Azure.SignalR.ServiceConnection[26]
      The connection L8g2QG7jU6iwwgPOCEUrDg1d404c941 has a long running application logic that prevents the connection from complete.
```

### Further technical details
- Your Azure SignalR SDK version: 1.9.1
- Your Server ASPNETCORE version 5.0.301
- Your SignalR Client SDK version 5.0.7

**Comment by vicancy** [comment]
Thanks for reporting the issue, investigating...

**Comment by vicancy** [comment]
Fixed in 1.9.2, thanks!
